ocs_ci/ocs/ocp.py

```diff
@@ -696,3 +697,69 @@ def rsync(src, dst, node, dst_node=True, extra_params=""):
         except CommandFailed:
             log.warning(f"Pod {pod_name} wasn't successfully deleted!")
             raise
+
+
+def get_images(data, images=None):
+    """
+    Get the images from the ocp object like pod, CSV and so on.
+
+    Args:
+        data (dict): Yaml data from the object.
```

Comment:
since we accept list of dict in code, can we add same in docstring

Reply:
Initial call of the function expect dictionary and cause of the recurse over whole dictionary it allow also go over the lists under the dictionary. As the user should pass the whole dictionary I would keep in the docstring the dict as this what which type the user is is suppose to call it.